basic-usage/configuration.md

```diff
@@ -79,18 +100,26 @@ FiddlerApplication.Prefs.SetStringPref("fiddler.proxy.pacfile.text", "return 'PR
 ```
 
 #### FiddlerCore proxy settings:
+
 - `ChainToUpstreamGateway()`: Sets the current LAN connection's proxy settings as an upstream gateway proxy. For example, if the application is running in a corporate environment behind a corporate proxy, the corporate proxy will be used as an upstream gateway proxy for FiddlerCore.
 >note Chaining when upstream gateway proxy settings are configured to use PAC file is supported only on Windows.
 - `SetUpstreamProxySettingsTo(ProxySettings)`: Sets the FiddlerCore upstream proxy settings as specified in the passed instance for ProxySettings class.
 
 #### Other settings:
+
+- `EnableHTTP2()`: Enables the support for capturing HTTP/2 traffic. Available with FiddlerCore version 6.x.x and above.
 - `DecryptSSL()`:  Enables decryption of HTTPS traffic. You should have a CertificateProvider loaded with trusted certificate. For more details see [Use Custom Root Certificate]({%slug use-custom-root-certificate %}) article.
-- `OptimizeThreadPool()`: Optimizes the thread pool to better handle multiple simultaneous threads. This is often convenient, as each `Session` is handled in a different thread. Under the hood, this methods uses [`ThreadPool.SetMinThreads`](https://docs.microsoft.com/en-us/dotnet/api/system.threading.threadpool.setminthreads) with a value larger that the default one.
+- `UseClientTlsProvider(customProvider)`: Enables you to set custom TLS provider as specfiied in the passed instance of type (IClientTlsConnectionProvider)[https://docs.telerik.com/fiddlercore/api/fiddler.iclienttlsconnectionprovider]. Available with FiddlerCore version 6.x.x and above.
+- `OptimizeThreadPool(custom)`: Optimizes the thread pool to better handle multiple simultaneous threads. This is often convenient, as each `Session` is handled in a different thread. Under the hood, this methods uses [`ThreadPool.SetMinThreads`](https://docs.microsoft.com/en-us/dotnet/api/system.threading.threadpool.setminthreads) with a value larger that the default one.
```

Comment:
This setting is no longer available. See bellow for a list of all available settings:
```
// The port on which the FiddlerCore app will listen on. If 0, a random port will be used.
ListenOnPort(ushort port)

// Registers as the system proxy.
RegisterAsSystemProxy()

// Decrypts HTTPS Traffic.
DecryptSSL()

// Enable support for capturing HTTP/2 sessions.
EnableHTTP2()

// Accepts requests from remote computers or devices. WARNING: Security Impact
AllowRemoteClients()

// Forwards requests to any upstream gateway.
ChainToUpstreamGateway()

// Sets all connections to use FiddlerCore, otherwise only the Local LAN is pointed to FiddlerCore.
MonitorAllConnections()

// Sets connections to use a self-generated PAC File.
HookUsingPACFile()

// Passes the <-loopback> token to the proxy exception list.(obsolete)
CaptureLocalhostTraffic()

// Registers FiddlerCore as the FTP proxy.(obsolete, do not use)
CaptureFTP()

// Sets the proxy settings which FiddlerCore uses to find the upstream proxy.
SetUpstreamProxySettingsTo(ProxySettings proxySettings)

// Sets the custom client TLS provider for Fiddler. The provider will be used to authenticate an existing connection and return  a stream to read/write data from/to it.
UseClientTlsProvider(IClientTlsConnectionProvider customClientTlsProvider)
```